src/routes.js

```diff
@@ -1,14 +1,29 @@
 import React from 'react';
-import { Route, Router, browserHistory } from 'react-router';
+import { Route, Router, IndexRoute, browserHistory } from 'react-router';
+import { syncHistoryWithStore } from 'react-router-redux';
 
+import Layout from 'views/Layout';
 import Home from 'views/Home';
 import Basic from 'containers/Basic';
 import Styleguide from 'views/Styleguide';
+import Petition from 'containers/Petition';
 
-export default (
-  <Router history={browserHistory}>
-    <Route path='/' component={Home} />
-    <Route path='/basic' component={Basic} />
-    <Route path='/styleguide' component={Styleguide} />
-  </Router>
-);
+export default function (props = {}) {
+  let history = browserHistory;
+
+  if (props.store) {
+    history = syncHistoryWithStore(browserHistory, props.store);
+  }
+
+  return (
+    <Router history={history}>
+      <Route path='/' component={Layout}>
```

Comment:
This route wraps our app, and provides us with a way to add header nav, footer etc.